quickstart-template/mass-payout.js

```diff
@@ -66,12 +66,15 @@ async function sendMassPayout(sendingWallet) {
       const address = row[0];
       if (address) {
         try {
-          await sendingWallet.createTransfer({
+          const transfer = await sendingWallet.createTransfer({
             // Send payment to each Address in CSV.
             amount: transferAmount,
             assetId: assetId,
             destination: address,
           });
+
+          await transfer.wait();
```

Comment:
this is a place where we can show how you can proceed with other transfers even if one fails with a try catch around the wait alone

Reply:
We should distinguish between the transfer failing on-chain (of which it will not throw an error currently) and timing out.

And have explicit handling of both scenarios.

I think that we need to do a pretty significant re-structuring of most of these examples in any case